script/release.sh

```diff
@@ -19,14 +19,51 @@ set -e
 # Project-specific options and functions. In *theory* you shouldn't need to
 # touch anything else in this script in order to use this elsewhere.
 project="runc"
-root="$(readlink -f "$(dirname "${BASH_SOURCE}")/..")"
+root="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")/..")"
 
 # This function takes an output path as an argument, where the built
 # (preferably static) binary should be placed.
 function build_project() {
+	local builddir
 	builddir="$(dirname "$1")"
 
-	make -C "$root" COMMIT_NO= static
+	# Due to libseccomp being LGPL we must include its sources,
+	# so download, install and build against it.
+
+	# Sanity check: check that the build won't use distro-provided
+	# libseccomp.a. If this is the case, explain and abort.
+	#
+	# Note go is not used here as it caches the build heavily,
+	# including, apparently, the results of pkg-config.
+	# shellcheck disable=SC2046
+	if read -ra flags < <(pkg-config --libs --cflags libseccomp 2>/dev/null) &&
+		cat <<EOF | gcc -static -x c -o /dev/null - "${flags[@]}"; then
+#include <seccomp.h>
+int main(void) { seccomp_version(); return 0; }
+EOF
+		set +x
+		echo >&2
+		echo "Distro-provided libseccomp static library is installed." >&2
+		echo "Unable to build a static binary against own libsecomp." >&2
+		echo "Please uninstall libseccomp-static to fix." >&2
+		exit 1
+	fi
+
+	local libseccomp_ver='2.5.1'
+	local tarball="libseccomp-${libseccomp_ver}.tar.gz"
+	local prefix
+	prefix="$(mktemp -d)"
+	curl -O -L "https://github.com/seccomp/libseccomp/releases/download/v${libseccomp_ver}/${tarball}"
```

Comment:
In order make my life even easier when I do releases, can you also download the `.asc` signature and then include both the tarball and the detached signature in the release directory so that they're part of the signed `sha256sum` and I can more easily upload the release artefacts?

Reply:
Thought about it but wasn't sure.

Added.